src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule231filetab/InputWhitespaceCharacters.java

```diff
@@ -5,58 +5,60 @@ final class InputWhitespaceCharacters {
   // Contains a tab ->	<- // violation 'Line contains a tab character.'
   // Contains trailing whitespace ->
 
-  /** Invalid format * */
-  public static final int badConstant = 2;
+  /** Invalid format. * */
+  public static final int badconstant = 2;
```

Comment:
what is reason to change name ?
soundslike you lowercased all as bulk update. If yes, I am ok.

please make javadoc ` /** Invalid format. */` (apply same to all)

Reply:
ok but I have a question about this code: https://github.com/checkstyle/checkstyle/pull/15352#issuecomment-2248443683

do we need this code?

Reply:
I think we can remove. Up to you.